src/main/java/org/folio/rest/impl/PermissionUtils.java

```diff
@@ -86,10 +86,9 @@ public static boolean equals(OkapiPermission okapiPerm, Permission perm) {
           List<Future<RowSet<Row>>> futures = new ArrayList<Future<RowSet<Row>>>();
           res.result()
             .forEach(row -> {
-              String id = row.getString("id");
               String name = row.getString("name");
-              futures.add(Future.<RowSet<Row>>future(p -> pgClient.execute(tx, String.format(PURGE_DEPRECATED_SUB_PERMS, tenantId, id, name, id, name), p)));
-              futures.add(Future.<RowSet<Row>>future(p -> pgClient.execute(tx, String.format(PURGE_DEPRECATED_PERMS_USERS, tenantId, id, name, id, name), p)));
+              futures.add(Future.<RowSet<Row>>future(p -> pgClient.execute(tx, String.format(PURGE_DEPRECATED_SUB_PERMS, tenantId, name, name), p)));
+              futures.add(Future.<RowSet<Row>>future(p -> pgClient.execute(tx, String.format(PURGE_DEPRECATED_PERMS_USERS, tenantId, name, name), p)));
               futures.add(Future.<RowSet<Row>>future(p -> pgClient.execute(tx, String.format(PURGE_DEPRECATED_PERMS, tenantId), p)));
```

Comment:
String.format should be replaced by the PostgresClient.execute method that takes `Tuple params`.

Reply:
Changed to use `Tuple`. I should have done that initially.